processing/src/main/java/io/druid/segment/incremental/IncrementalIndex.java

```diff
@@ -758,6 +758,22 @@ public String getMetricType(String metric)
     return metricDesc != null ? metricDesc.getType() : null;
   }
 
+  public Class getMetricClass(String metric)
```

Comment:
@navis can you update the master comment of this PR to indicate why this change is needed and how this solves a blocking issue for the extension?